ProjFS.Linux/PrjFSLib.Linux.Managed/VirtualizationInstance.cs

```diff
@@ -231,74 +221,65 @@ private static string GetProcCmdline(int pid)
         }
 
         // TODO(Linux): replace with netstandard2.1 Marshal.PtrToStringUTF8()
```

Comment:
Is there anything left to do for this TODO?

`// TODO(Linux): replace with netstandard2.1 Marshal.PtrToStringUTF8()`

Reply:
Not really ... I mused over whether to just call `Marshal.PtrToStringAnsi()` in the three places we're calling our wrapper, and left it in place for now only because I think (and may be wrong!) that if we ever build on .NET Standard 2.1, we could explicitly call `Marshal.PtrToStringUTF8()`, which would be a little cleaner to read.

It looks like the `PtrToStringUTF8()` was only [introduced for .NET Standard 2.1](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.marshal.ptrtostringutf8?view=netstandard-2.1), but we're using only [.NET Standard 2.0](https://github.com/Microsoft/VFSForGit/blob/features/linuxprototype/ProjFS.Linux/PrjFSLib.Linux.Managed/PrjFSLib.Linux.Managed.csproj#L3) right now.

Reply:
I don't have a strong preference here, I just wasn't sure with the change to call a `Marshal` method if you still wanted to leave the `TODO` comment in place.  

If it's a reminder to change methods when we eventually update to 2.1 I'm good with leaving it.

Reply:
I'm also sort of hoping/expecting that the C# compiler or runtime will optimize away the wrapper.